doc/api/process.md

```diff
@@ -1996,6 +1996,129 @@ Will generate an object similar to:
   tz: '2016b' }
 ```
 
+## Access Control
+
+> Stability: 1 - Experimental
+
+### process.accessControl.apply(restrictions)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `restrictions` {Object} A set of access restrictions that will be applied
+  to the current Node.js instance. The format should be the same as the one
+  returned by [`process.accessControl.getCurrent()`][]. Omitted keys will
+  default to `true`, i.e. to retaining the relevant permissions.
+
+*Warning*: This does not provide a full isolation mechanism. Child processes,
+native addons, and other communication channels may be used to circumvent
+these measures, if they have been made available before the corresponding
+restrictions have been put into place.
+
+This API is recent and may not be complete.
+Please report bugs at https://github.com/nodejs/node/issues.
+
+Operations scheduled before this call are not undone or stopped.
```

Comment:
"are not...stopped" -- this smells like a race to me. It looks like they *may* be stopped, if for example I read a file and then load a module, and before the read completes I set access controls.

Reply:
Hm… It’s not a race, though, is it? If you read a file and then set access controls, the read is always going to finish like it usually would…

Does this sentence sound like that’s not the case?

Reply:
```
fs.readFile
.then(
  fs.readFile
);

process.accessControl.apply( { fsRead: false } );
```

The first read will finish. The second read, which is scheduled (or is it? maybe that's the word I'm confused by.), will not finish in this case.

Reply:
Ah, yes. For me “scheduled” meant that the the call already occurred, just wasn’t finished. Is “started” a better word? (I’m switching over to that but please make other suggestions if you have any. :))